src/services/ImageTransforms.php

```diff
@@ -428,6 +428,8 @@ public function eagerLoadTransforms(array $assets, array $transforms): void
                         ...$refTransform->toArray(),
                 ]);
 
+                unset($transform->name, $transform->handle);
```

Comment:
This bit probably isn't strictly necessary, assuming you'll eventually be going through `Asset::getUrlsBySize`. However, ensuring `name` and `handle` aren't set when manipulating the transform seems like a sensible precaution to keep behavior unchanged by https://github.com/craftcms/cms/pull/15646